### ISSUE TITLE
HTTP Error 500 - Export Manager Issue

### ISSUE THREAD

**Issue by nicktalsma** [formtools/core#500]
Recently upgraded from 3.0.2 to 3.0.13. 

I am getting a HTTP 500 error when exporting large quantities of records from the form to Excel.  I tried enabling CSV and I got the same error. 

The odd thing is, if I scale back my export size, from say 1,000 records to 500 records, it works.  

Any ideas?

**Comment by benkeen** [comment]
Hey @nicktalsma, hmm, sounds like it's either a memory issue or the code is just timing out. 

I checked the timeout value and see it's hardcoded at 5 mins. Does it take that long to show the 500 error, or is it much quicker than that? (in which case it'll be memory).

For memory, you'll need to contact your hosting provider to increase the memory limit in your server's php.ini file (or possibly your env may allow you to do it yourself).

Bottom line is that generating large files/data sets is always going to be a memory hog so currently it'll be a problem on all servers - it's just a question of available memory. Sorry about that.
